validation/checks/ark_check.go

```diff
@@ -13,7 +11,6 @@ import (
 
 	"go.uber.org/multierr"
 )
-
 // ITEM_ARK is the ARK of the current item.
 const ITEM_ARK = "Item ARK"
```

Comment:
Not requesting any changes now (I can create a separate ticket later), but I've since learned idiomatic Go uses CamelCase for const too (so my bringing over the all-caps syntax from Java was a mistake). Just noting, but not requesting any changes in this PR.